pkg/entity/datapolicy/datapolicy.go

```diff
@@ -143,6 +160,16 @@ func readPolicy(filename string) *DataPolicy {
 	return dataPolicy
 }
 
+func addSampleDataFlag(cmd *cobra.Command, flags *pflag.FlagSet) {
+	flags.String(common.SampleDataFlag, "", common.SampleDataUsage)
+	CliExit(
+		cmd.RegisterFlagCompletionFunc(common.SampleDataFlag,
+			func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+				return []string{"csv,txt"}, cobra.ShellCompDirectiveFilterFileExt
```

Comment:
Why complete on `txt`? Do you think that many people would suffix their `csv` content with `txt` in the filename?